src/pudl/metadata/fields.py

```diff
@@ -1485,7 +1485,7 @@
     "row_type_xbrl": {
         "type": "string",
         "description": "Indicates whether the value reported in the row is calculated, or uniquely reported within the table.",
-        "constraints": {"enum": ["calculated", "ferc_account"]},
+        "constraints": {"enum": ["calculated", "ferc_account", "energy_source"]},
```

Comment:
i'm not sure if we'll want to maintain a very long list of row types... if this list gets big. but added this here for now... maybe this hsould just be `reported`?

Reply:
these values are coming in through some very simple assigns in the xbrl metadata normalization. i like these three reported/calculated/header values (even though the headers don't how up in the metadata)

Reply:
for the pis table you flagged when something is a ferc_account
```
        pis_meta.loc[
            ~pis_meta.calculations.astype(bool) & pis_meta.ferc_account.notna(),
            "row_type_xbrl",
        ] = "ferc_account"
```

which.... is helpful but not that helpful bc you can always just check the `ferc_account` column. would you feel okay about me moving to standardize this as well and take away `ferc_account`?

Reply:
After talking about this, it seems like this should just be the `calculated_value` and `reported_value` categories, since we don't want to enumerate all of the different possible types of values here -- if anything we'll extract that information from the XBRL metadata, once we're processing it more completely.

Reply:
In that meta flagging, I think I was looking at the actual ferc account string from the metadata, wasn't I? Like "408.1"?